Sources/MapboxCoreNavigation/EHorizon/ElectronicHorizonEdgeMetadata.swift

```diff
@@ -0,0 +1,83 @@
+import Foundation
+import CoreLocation
+import MapboxDirections
+import MapboxNavigationNative
+
+extension ElectronicHorizon.Edge {
+
+    /** Edge metadata */
+    public struct Metadata {
+
+        /** The bearing in degrees clockwise at the start of the edge. */
+        public let heading: CLLocationDegrees
+
+        /** The Edge's length in meters. */
+        public let length: CLLocationDistance
+
+        /** The edge's functional road class */
+        public let mapboxStreetsRoadClass: MapboxStreetsRoadClass
+
+        /** The edge’s maximum speed limit. */
+        public let speedLimit: Measurement<UnitSpeed>?
+
+        /** The Edge's average speed (m/s) */
+        public let speed: CLLocationSpeed
+
+        /** Is the edge a ramp? */
+        public let isRamp: Bool
+
+        /** Is the edge a motorway? */
+        public let isMotorway: Bool
+
+        /** Is the edge a bridge? */
+        public let isBridge: Bool
+
+        /** Is the edge a tunnel? */
+        public let isTunnel: Bool
+
+        /** Is the edge a toll road? */
+        public let isToll: Bool
+
+        /** The edge's names */
+        public let names: [RoadName]
+
+        /** The edge's lane counts */
+        public let laneCount: UInt?
+
+        /** The edge's mean elevation */
+        public let meanElevation: CLLocationDistance?
+
+        /** The edge's curvature */
+        public let curvature: UInt
```

Comment:
What unit is the curvature measured in?

Reply:
It is "curvature factor" https://github.com/mapbox/valhalla-internal/blob/c4799e97e0721742101b244c8bf771653b57edb8/valhalla/baldr/directededge.h#L227 . I  guess there are no any units.